content/advisories/2021-11-11-transaction-broadcast.zh-tw.md

```diff
@@ -0,0 +1,28 @@
+
++++
+title = "公告：對交易自動重新進行的回應"
+date = "2021-11-11"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+幣安今日暫停了狗狗幣的提款功能，並指出他們發現狗狗幣的＂小問題＂。對此我們想做出回應：
+
+數個月前 (請注意，在稍早發文中說的是一年前，但首次確認的提及是在四月) 幣安告知我們他們遇到一些交易卡住的情況，交易無法成功被礦工打包。我們建議幣安對這些交易使用 RBF (Replace-By-Fee) 機制 - 以使用較高手續費的新交易取代原先的交易。值得重視的是，RBF 能使先前的交易失效（因此也可說是＂取代＂），所以這個作法被優先建議。由於那些交易沒有啟用 RBF，我們建議手動建立使用相同 input 的新交易，強制使原先的交易失效。
+
+一段時間之後，幣安向我們表示他們遇到帳目核對問題。我們無法用幣安提供的資料重現那些狀況，但我們（在距今的幾個月前）建議了使用 `-zapwallettxes` 指令。值得注意的是，我們相信有使用此作法也就能防止該問題。
```

Comment:
Some time later Binance notified **the maintainers** that they had account reconciliation issues. **The maintainers** were unable to reproduce those issues with the data we were given by Binance, but at the time suggested using the -zapwallettxes command line option to mitigate the issue. This is notable as it is anticipated this would have also prevented the issue seen.

-----

lease correct the highlighted sentences as above in the translation, according to the latest updates.